runtime/Java/src/org/antlr/v4/runtime/misc/IntegerList.java

```diff
@@ -268,6 +267,8 @@ public final int binarySearch(int fromIndex, int toIndex, int key) {
 		if (fromIndex < 0 || toIndex < 0 || fromIndex > _size || toIndex > _size) {
 			throw new IndexOutOfBoundsException();
 		}
+        if (fromIndex > toIndex)
```

Comment:
I think this condition makes sense. Can you update the code to match the surrounding formatting (hard tabs, and use braces)?